### ISSUE TITLE
[BUG] error message duplicated for command `rudr system:info`

### ISSUE THREAD

**Issue by zzxwill** [oam-dev/kubevela#125]
```
[root@iZj6cc54ecpc3pm8t9yctuZ Rudrx]# vela system:info
fail to get cluster version, err: oam-kubernetes-runtime not found in your kubernetes cluster, try `vela system:init` to install.
Error: oam-kubernetes-runtime not found in your kubernetes cluster, try `vela system:init` to install.
```

**Comment by wonderflow** [comment]
please help fix this minor issue， thanks

**Comment by zzxwill** [comment]
Fixed in #303 by @mosesyou .
